recipes/astrometry/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set name = "astrometry" %}
+{% set version = "0.78" %}
+
+package:
+  name: {{ name|lower}}
+  version: {{ version }}
+
+source:
+  fn: {{name}}-{{ version }}.zip
+  url: https://github.com/dstndstn/astrometry.net/archive/{{ version }}.zip
+  sha256: 85480b9636757c89ef5ccc7aac557409ac2ab377eb6c6eca862b16cdd4faefd9
+
+build:
+  number: 0
+  skip: True  # [osx or win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - swig
+    - make
+  host:
+    - xorg-libx11  # [linux]
+    - xorg-libxau  # [linux]
+    - xorg-libxext  # [linux]
+    - xorg-libxrender  # [linux]
+    - xorg-xproto  # [linux]
+    - numpy
+    - zlib
+    - jpeg
+    - netpbm
+    - libpng
+    - cfitsio
+    - astropy
+    - bzip2
+    - cairo
+    - gsl
+  run:
+    - zlib
+    - jpeg
+    - netpbm
+    - libpng
+    - cfitsio
+    - astropy
+    - numpy
+    - bzip2
+    - cairo
+    - gsl
+
+test:
+  commands:
+    - solve-field -h
+
+about:
+  home: http://astrometry.net/
+  license: BSD
```

This should state which licence it is (replace N as appropriate):
```suggestion
  license: BSD-N-Clause
  license_family: BSD
```